yarn.lock

```diff
@@ -2,13 +2,6 @@
 # yarn lockfile v1
 
 
```

Comment:
I would recommend to add this file to .gitignore files list. @rogergraves what do you think?

Reply:
Yes, agreed

Reply:
Thanks guys)